views/profile.ejs

```diff
@@ -0,0 +1,72 @@
+<form action="/profile" id="profile" method="post" name="profile" role="form">
+  <fieldset>
+    <legend>Utilisateur</legend>
+    <div class="form-group">
+      <label for="cip">CIP:</label>
+      <input type="text" class="form-control" id="cip" name="cip" value="<%= userInfo.cip %>" readonly>
+    </div>
+    <div class="form-group">
+      <label for="prenom">Prénom:</label>
+      <input type="text" class="form-control" id="prenom" name="prenom" pattern="<%= regExes.name %>" value="<%= userInfo.prenom%>" required>
+    </div>
+    <div class="form-group">
+      <label for="nom">Nom:</label>
+      <input type="text" class="form-control" id="nom" name="nom" pattern="<%= regExes.name %>" value="<%= userInfo.nom %>" required>
+    </div>
+    <div class="form-group">
+      <label for="email">Adresse courriel:</label>
+      <input type="email" class="form-control" id="email" name="email" pattern="<%= regExes.email %>" placeholder="adresse@USherbrooke.ca ou @AGEG.ca" title="Adresse courriel @USherbrooke.ca ou @AGEG.ca" value="<%= userInfo.email %>" required>
+    </div>
+    <div class="form-group">
+      <label for="phone">Téléphone:</label>
+      <input type="tel" class="form-control" id="phone" name="phone" pattern="<%= regExes.phone %>" placeholder="ex: 1-819-569-3776" title="Numéro de téléphone à 9 ou 10 chiffres. Les délimiteurs optionnels autorisés sont '(' ,')' , '-' et ' '." value="<%= userInfo.phone %>" required>
+    </div>
+  </fieldset>
+    <div class="form-group">
+      <label for="concentration">Concentration:</label>
+      <select class="form-control" id="concentration" name="concentration">
+        <% if(typeof userInfo != 'undefined' && userInfo.concentration.localeCompare('Biotechnologique') === 0) { %>
```

Comment:
This could make made using an array (probably exposed by the mongoose model) and a for loop. Where you just to a check if it's selected, and if it is you render it as selected.